util/Setup/Templates/DockerCompose.hbs

```diff
@@ -220,6 +193,4 @@ volumes:
 
 networks:
   default:
-    internal: true
-  public:
-    internal: false
```

Comment:
The original idea here was to have an internal network for containers that don't need to access WAN. That isolates them even more. For example, SQL Server tries to send out telemetry to MS and this will block that.

Reply:
I didn't realize the SQL Server had telemetry.  I reverted the template to what it was before.